### DISCUSSION TITLE
Can I do some calculations like efficiency, power input and output but manipulating the air flow and pressure?

### DISCUSSION THREAD

**Discussion by blower05** [ProjectPhysX/FluidX3D#49] [Q&A] [answered]
I have an impeller design in mind for years but got no resource to validate the design. 
Is FluidX3D capable of calculations like efficiency, power input and output but manipulating the air flow and pressure?
Thanks!

**Comment by ProjectPhysX** [comment, marked as answer]
Yes, very soon. I'm currently working on re-voxelization of moving boundaries. Code is done, now validation. I'll push this in update v2.2 later this week. Right now I'm running a simulation of an electric ducted fan on my PC.

**Reply by blower05** [reply]
wow! awesome but it looks there is no GUI for Fluidx3d. I tried learning openFOAM before but really stuck at some errors during to too many variations and versions of it. Finally, I still can't solve it at it spent long time in solvers as my computer is too slow. I have big Shadow on this interface and time consuming issues.
